### ISSUE TITLE
[BUG] Missing v-bind attributes in SfHeader

### ISSUE THREAD

**Issue by krskibin** [DivanteLtd/storefront-ui#1433]
**Describe the bug**
Missing v-bind attributes `wishlistHasProducts, wishlistItemsQty` in SfHeader.vue

**Comment by Baroshem** [comment]
Hey @krskibin 

Could you please explain in more details what do you mean that the attributes are missing? I checked SfHeader they seem to be there.


```
// line 192
    /**
     * Header wishlist items quantity
     */
    wishlistItemsQty: {
      type: [String, Number],
      default: "0",
    },
```

wishlistHasProducts -> it is a computed property
```
// line 238
    wishlistHasProducts() {
      return parseInt(this.wishlistItemsQty, 10) > 0;
    },
```

I tested this on both nuxt and develop branch and they both seem to be working correctly. What does not work are the badges that should appear when there are items in the wishlist or cart but this was explained here -> https://github.com/vuestorefront/storefront-ui/pull/1514#issuecomment-755972554

**Comment by krskibin** [comment]
Yes, all good. This ticket is kinda old, so it probably was fixed in another issue/pr.

**Comment by AdamPawlinski** [comment]
I think it was added in another PR so we can close this issue.
